extension/defaults.js

```diff
@@ -1,9 +1,9 @@
 const storage = getApi().storage.local;
 //default values for options.
 const defaults = {
-	organization_domain: "",
-	google_spid: "",
-	google_idpid: "",
+	organization_domain: "studocu.com",
+	google_spid: "308937699935",
+	google_idpid: "C00yxev13",
```

Comment:
```suggestion
	organization_domain: "",
	google_spid: "",
	google_idpid: "",
```

These are secret values, and this repo is public :grimacing: